src/main/java/com/comphenix/protocol/injector/StructureCache.java

```diff
@@ -152,6 +152,7 @@ private static ConstructorAccessor getTrickDataSerializerOrNull() {
 				// can happen if unsupported
 			}
 		}
-		return TRICKED_DATA_SERIALIZER;
+
```

Comment:
```suggestion
```

Reply:
I dont really know if I can agree with that since this is really just down to readability and imo this makes the code far more readable